provider-registry.yaml

```
@@ -0,0 +1,2 @@
+- git+https://github.com/i-am-bee/beeai@main#path=/agents/community/gpt-researcher-agent
+- git+https://github.com/i-am-bee/beeai@main#path=/agents/official/bee-agent-framework
```

Maybe we want to go a bit more verbose in case we need to extend later (add more config keys)?

```suggestion
providers:
- url: git+https://github.com/i-am-bee/beeai@main#path=/agents/community/gpt-researcher-agent
- url: git+https://github.com/i-am-bee/beeai@main#path=/agents/official/bee-agent-framework
```